pkg/alertmanager/multitenant.go

```diff
@@ -67,11 +66,11 @@ const (
 )
 
 var (
-	totalConfigs = prometheus.NewGauge(prometheus.GaugeOpts{
+	totalConfigs = prometheus.NewGaugeVec(prometheus.GaugeOpts{
 		Namespace: "cortex",
 		Name:      "alertmanager_configs",
 		Help:      "How many configs the multitenant alertmanager knows about.",
-	})
+	}, []string{"status"})
```

Comment:
I would mention this change in the changelog too. What do you think?

Reply:
Will do!